yaml/xyz/openbmc_project/Inventory/Item/PCIeDevice.interface.yaml

```diff
@@ -287,3 +287,18 @@ properties:
       type: string
       description: >
           The name of the Manufacturer for this device.
+
+    - name: LanesInUse
```

Comment:
What does this map to in ASMI? Link Width? 
https://github.ibm.com/gmills/pcie-topology

            row["Link Status"] = "Open"
            row["Link Type"] = "Primary"
            row["Link Speed"] = "Unknown"
            row["Link Width"] = "Unknown"

Reply:
It maps to Link Width.